kong.config.sh

```diff
@@ -70,7 +70,19 @@ PAYLOAD
 PAYLOAD
 authConfig "device-manager"
 
-# auth service configuration
+
+(curl -o /dev/null ${kong}/apis -s -S -X POST \
+    --header "Content-Type: application/json" \
+    -d @- ) <<PAYLOAD
+{
+    "name": "image",
+    "uris": "/fw-image",
```

Comment:
using this as opposed to "/image" because that would break GUI (already uses that prefix)